plugin/build.gradle

```diff
@@ -322,7 +322,7 @@ jacocoTestCoverageVerification {
 check.dependsOn jacocoTestCoverageVerification
 
 configurations.all {
-    resolutionStrategy.force 'org.apache.commons:commons-lang3:3.10'
+    resolutionStrategy.force 'org.apache.commons:commons-lang3:3.13.0'
```

Comment:
Do we still need to force the version if we already use `"${versions.commonslang}"` to get the version dynamically?